ui/src/tag_table_cell_type.tsx

```diff
@@ -66,7 +68,7 @@ export const XTagTableCellType = ({ model, serializedTags }: { model: TagTableCe
         background = cssVar(tagColor),
         color = cssVar(tag?.label_color || getContrast(tagColor))
 
-      return <span key={i} style={{ background, color }} className={clas(css.tag, 'wave-s12 wave-w6')}>{tagLabel}</span>
+      return <div key={i} style={{ background, color }} className={clas(css.tag, 'wave-s12 wave-w6', isMultiline ? css.multiline : '')}>{tagLabel}</div>
```

Comment:
Why do we need `div` instead of `span` here?

Reply:
It was helpful in debugging process, but with current solution it's not necessary anymore. Replacing back with `span`.